scripts/auto-rebase/rebase.sh

```diff
@@ -891,31 +1035,38 @@ rebase_to() {
 
 usage() {
     echo "Usage:"
-    echo "$(basename "$0") to RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM         Performs all the steps to rebase to a release image. Specify both amd64 and arm64."
-    echo "$(basename "$0") download RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM   Downloads the content of a release image to disk in preparation for rebasing. Specify both amd64 and arm64."
-    echo "$(basename "$0") buildfiles                                       Updates build files (Makefile, Dockerfile, .spec)"
-    echo "$(basename "$0") go.mod                                           Updates the go.mod file to the downloaded release"
-    echo "$(basename "$0") generated-apis                                   Regenerates OpenAPIs"
-    echo "$(basename "$0") images                                           Rebases the component images to the downloaded release"
-    echo "$(basename "$0") manifests                                        Rebases the component manifests to the downloaded release"
+    echo "$(basename "$0") to RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM LVMS_RELEASE_MULTI_ARCH        Performs all the steps to rebase to a release image. Specify both amd64 and arm64 ocp releases, and multi-arch lvms operator bundle image."
```

Comment:
So, job will have this argument hardcoded to `registry.redhat.io/lvms4/lvms-operator-bundle:v4.YY` depending on branch it runs for?  Like `release-4.12` -> `v4.12`, `main` -> `v4.13` (where it's released)?